keyboards/vitamins_included/vitamins_included.h

```diff
@@ -1,10 +1,37 @@
-#ifndef VITAMINS_INCLUDED_H
-#define VITAMINS_INCLUDED_H
+#pragma once
 
 #include "quantum.h"
 
-#include "rev1.h"
+#ifdef KEYBOARD_vitamins_included_rev1
+    #include "rev1.h"
+#else
+    #include "rev2.h"
+#endif
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+	#include <avr/io.h>
+	#include <avr/interrupt.h>
+#endif
+#endif
 
+#define KEYMAP( \
+	L00, L01, L02, L03, L04, L05, R00, R01, R02, R03, R04, R05, \
+	L10, L11, L12, L13, L14, L15, R10, R11, R12, R13, R14, R15, \
+	L20, L21, L22, L23, L24, L25, R20, R21, R22, R23, R24, R25, \
+	L30, L31, L32, L33, L34, L35, R30, R31, R32, R33, R34, R35 \
+	) \
+	{ \
+		{ L00, L01, L02, L03, L04, L05 }, \
+		{ L10, L11, L12, L13, L14, L15 }, \
+		{ L20, L21, L22, L23, L24, L25 }, \
+		{ L30, L31, L32, L33, L34, L35 }, \
+		{ R00, R01, R02, R03, R04, R05 }, \
+		{ R10, R11, R12, R13, R14, R15 }, \
+		{ R20, R21, R22, R23, R24, R25 }, \
+		{ R30, R31, R32, R33, R34, R35 } \
+  }
 
 // Used to create a keymap using only KC_ prefixed keys
 #define KC_KEYMAP( \
```

Comment:
This macro should be named `LAYOUT_kc`, and Line 43 should be updated to use `LAYOUT`.

```suggestion
#define LAYOUT_kc( \
```